raffle.py

```diff
@@ -1,10 +1,15 @@
 import csv
 import random
-import sys
+import argparse
 from random import shuffle
-totalWinners = 48
+parser = argparse.ArgumentParser()
+parser.add_argument("csv",help = "the location of the csv input")
+parser.add_argument("totalWinners", type=int, help="total number of selected winners")
+args = parser.parse_args()
 winlist = []
-with open(sys.argv[1]) as csvfile:
+csvrow = open(args.csv)

```

Comment:
You never use this variable? And also, not really sure what this var is supposed to represent.